NEWS.md

```diff
@@ -3,6 +3,13 @@
 v0.2.9
 -------
 
+## Features
+
+### Add a `start_date` attribute to NetCDFWriter PR [#94](https://github.com/CliMA/ClimaDiagnostics.jl/pull/94).
+
+Prior to this version, users had to go to the simution to find the start date.
+It is now saved as an attribute, making it easily accessible.
```

Comment:
Can you describe how to use this new feature? It requires user interaction (adding a call to `start_date` in the constructor)